lib/components/narrative/metro/attribute-utils.tsx

```diff
@@ -0,0 +1,115 @@
+import { FormattedList, FormattedTime, IntlShape } from 'react-intl'
+import { Itinerary, Leg } from '@opentripplanner/types'
+import coreUtils from '@opentripplanner/core-utils'
+import React from 'react'
+
+import { firstTransitLegIsRealtime } from '../../../util/viewer'
+
+export const departureTimes = (
+  itinerary: Itinerary & {
+    allStartTimes: { arrival: number; realtime: boolean; time: number }[]
+  },
+  intl: IntlShape
+): JSX.Element => {
+  if (!itinerary.allStartTimes) {
+    return (
+      <span
+        className={
+          firstTransitLegIsRealtime(itinerary) ? 'realtime first' : 'first'
+        }
+        title={intl.formatMessage(
+          { id: 'components.MetroUI.arriveAtTime' },
+          { time: intl.formatTime(itinerary.endTime) }
+        )}
+      >
+        <FormattedTime value={itinerary.startTime} />
```

Comment:
itinerary.startTime is not always displayed. Sometimes just "leave now" shows up with no time

Reply:
Please send a screenshot the next time you see it! I am still unable to replicate this.